src/displayUsers.js

```diff
@@ -0,0 +1,53 @@
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
- [ ] It would be nice if you don't disable the linters and manually fix all the errors to make your app free from those errors which can break your app completely 😨